### PR TITLE
cmd ref: improvements to params and 1.0 update!

### PR DESCRIPTION
**Pull request by jorgeorpinel**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
The intention is that you get to the info. on how to create params faster (dvc run -p). I think that is more important than the other important explanations I moved under. There's really no other doc that covers param creation except perhaps a passage in Get Started.

**Comment by jorgeorpinel** [comment]
and the example in dvc run

**Comment by shcheklein** [comment]
But those two paragraphs look strange in "benefits and workflow" section

File: content/docs/command-reference/params/index.md [hunk 1]
```diff
@@ -23,41 +23,43 @@ version of the project, and defined (using `dvc run`) with simple names like
...
-
-Supported parameter _value_ types are: string, integer, float, and arrays. DVC
-itself does not ascribe any specific meaning for these values. They are
-user-defined, and serve as a way to generalize and parametrize an machine
-learning algorithms or data processing code.
+of a YAML or JSON _parameters file_ and a parameter name (key to find inside the
+parameters file). Multiple parameter dependencies can be specified from one or
+more parameters files.
 
 ### Benefits and workflow
```
Comment (shcheklein) [inline content/docs/command-reference/params/index.md:30]: may be just remove this title? it looks artificial

**Comment by shcheklein** [comment]
May be we can just remove that sub-section

**Comment by jorgeorpinel** [comment]
I removed it. I also reorganized paragraphs again (closer to the original).

I feel like this description is too long though. But that's kind of out of scope right now.

**Comment by jorgeorpinel** [comment]
Oops sorry just after I said the scope was limited I went ahead and updated this whole doc for 1.0!

**Pull request merged**